### DISCUSSION TITLE
Is created Skeletal Mesh Components suppose to shows in Editor Panel?

### DISCUSSION THREAD

**Discussion by ks-zealot** [AndreaCatania/RuntimeSkeletalMeshGenerator#22] [Q&A]
After some work I could update the plugin for Unreal 5. Unfortunately, this new created skeleton components dont shows in blueprint editor panel.
Is it as suppose to be in Unreal 4? Or this code work fine with BP in previous versions?

**Comment by matiasgql** [comment]
This is a plugin for work in C++, is not for Blueprints, you should make your own character actor and put the runtime skeletal mesh instead of the regular skeletal mesh to see in editor
